addons/mail/static/tests/owl/component/chatter_tests.js

```diff
@@ -15,9 +15,9 @@ QUnit.module('component', {}, function () {
 QUnit.module('Chatter', {
     beforeEach() {
         utilsBeforeEach(this);
-        this.createChatter = async ({ id, model }, otherProps) => {
+        this.createChatter = async ({ chatterLocalId }, otherProps) => {
```

Comment:
would rename helper `createChatterComponent` (to not confuse entity and component)